File: Aman/Arrays & strings/Valid Anagram.cpp
class Solution {
public:
    bool isAnagram(string s, string t) {
        int n=s.length(),m=t.length();
        int dat[26]={0};
        for(int i=0;i<n;i++)
            dat[s[i]-'a']++;
        for(int i=0;i<m;i++)
            dat[t[i]-'a']--;
        for(int i=0;i<26;i++)
            if(dat[i]!=0)
                return false;
        return true;
    }
};

File: Harish Kumar/Array and Strings/Valid Anagram/validAnagram.java
class Solution {
    public boolean isAnagram(String s, String t) {
        
        if(s.length() != t.length())
        {
            return false;
        }
        
        char a[] = s.toCharArray();
        char b[] = t.toCharArray();
      
        Arrays.sort(a);
        Arrays.sort(b);
        
        int i = 0;
        
        while(i < a.length)
        {
            if(a[i]== b[i])
            {
                i++;
            }
            else
            {
                return false;
            }
        }
        return true;
    }
}
